bats/tests/helpers/snapshots.bash

```diff
@@ -1,5 +1,5 @@
 delete_all_snapshots() {
-    run rdctl snapshot list --json
+    run --separate-stderr rdctl snapshot list --json
     assert_success
```

Comment:
I only see `rdctl snapshot list` potentially write to `STDERR` when running without the `--json` option (in `tabularOutput`).

I would argue that either this should not be considered an error and then "No snapshots present" should be written to `STDOUT` instead of `STDERR`. Or it is an error and then it should have been returned via `errors.New()` and the exit status would automatically be set to `1` by Cobra..

I can see arguments for both ways to handle this, but given this is for human consumption (we have `--json` for automation), I don't see the benefit of writing to `STDERR` here.

Either way, I don't see how the `--json` codepath can write to `STDERR` and not exit with an error.

If we don't expect to see any output on `STDERR` here at all, then I would again suggest the wrapper I've shown above because it also verifies that `$stderr` is empty:

```suggestion
    run_without_error rdctl snapshot list --json
```